docs/examples/userguide/extension_types/cheesy.pyx

```diff
@@ -0,0 +1,36 @@
+ 
+
+
```

Comment:
Oh, @matusvalo  .. that's so nice of you that you give more attention to the alignment, so cool!
Sorry, I couldn't resist making this comment 😅

Reply:
I have too big screen at home - looks better on them ;-)

Reply:
I am getting better every day.